src/index.css

```
@@ -642,14 +674,15 @@ html {
 		 */
-		@apply rounded-[6px] border border-line-strong p-6;
+		@apply rounded-[var(--fp-radius-lg)] border border-line-strong p-6;
 	}
 
 	.dark .card {
 		@apply border-line;
 	}
 	.page {
-		@apply flex flex-col px-4 py-12 h-screen;
+		@apply flex h-screen flex-col px-4 pb-12;
+		padding-top: var(--fp-page-pad-top);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)